README.md

```diff
@@ -43,5 +56,26 @@ func main() {
 
     fmt.Println(fastestGasPrice)
     fmt.Println(fastGasPrice)
+
+    // alternatively, use the NewGasPriceSuggester which maintains a cache of results until they are older than max age
+    suggestGasPrice, err := gas.NewGasPriceSuggester(5 * time.Second)
+    if err != nil {
+        log.Fatal(err)
+    }
+
+    fastGasPriceFromCache, err := suggestGasPrice(gas.GasPriorityFast)
+    if err != nil {
+        return nil, err
+    }
+
+    // after 5 seconds, the cache will be invalidated and new results will be fetched
+    time.Sleep(5 * time.Second)
```

Comment:
Any idea how frequently the price specified by the eth gas station changes? 5 seconds feels like a shorter amount of time than is realistic, but I'm curious to hear your reasoning.

Reply:
I've played around with it a bit, and my sense it that ETH Gas Station has some sort of load balancing, because if you hit the API multiple times in secession, the results seem to change each time (to a small degree) as if they are round-robining 3-5 different servers.

Significant changes _can_ happen every few minutes, but generally (anecdotally) they only change significantly on the order of 10-30 minutes. Of course that depends on network conditions.

Reply:
I updated the duration used in the example to 5 minutes since I think you're right, that's probably a better example of typical usage. Of course no one is going to actually run the example for any reason other than to try it out, but it's helpful nonetheless.

Reply:
Yep, I agree people won't use the example, but they may _copy_ it.